roles/rook-recovery/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+  - name: Include secrets
+    include_vars: "{{ secrets_path | mandatory }}/aws.yaml"
+    no_log: true
+    tags:
+      - always
+
+  - name: Launch instance
+    ec2:
+       aws_access_key: "{{ aws_access_key }}"
```

Comment:
Looks like there's one extra space under `ec2:` than there should be.

Reply:
fixed